guides/source/5_1_release_notes.md

```diff
@@ -359,7 +359,7 @@ Please refer to the [Changelog][action-mailer] for detailed changes.
 *   Allowed passing lambdas as values to the `default` method.
     ([Commit](https://github.com/rails/rails/commit/1cec84ad2ddd843484ed40b1eb7492063ce71baf))
 
-*   Added support for parameterized invocation of mailers to share before filters and defaults
+*   Added support for parameterized invocation of mailers to share before action callbacks and defaults
```

Comment:
Drop the 'before': it's falsely narrowing what we imply will work, and "before action callbacks" is a pretty unpleasant noun-phrase.